src/hdinsightonaks/azext_hdinsightonaks/aaz/latest/hdinsightonaks/cluster/_create.py

```diff
@@ -0,0 +1,1986 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "hdinsightonaks cluster create",
+    is_preview=True,
+)
+class Create(AAZCommand):
+    """Create a cluster.
+
+    :example: Create a Trino cluster.
+        az az hdinsightonaks cluster create -n clustername --cluster-pool-name clusterpoolname -g resourcesGroup -l westus3 --assigned-identity-object-id 00000000-0000-0000-0000-000000000000 --assigned-identity-client-id 00000000-0000-0000-0000-000000000000 --authorization-user-id 00000000-0000-0000-0000-000000000000 --assigned-identity-id /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/PSGroup/providers/Microsoft.ManagedIdentity/userAssignedIdentities/yourmsi --cluster-type Trino --cluster-version 1.0.6 --oss-version 0.410.0 --nodes (az hdinsightonaks cluster node-profile create  --count 5 --type Worker --vm-size Standard_D8d_v5)
```

Comment:
We recommend using [shorthand syntax](https://github.com/Azure/azure-cli/blob/dev/doc/shorthand_syntax.md) for complex value arguments or create [sub commands](https://azure.github.io/aaz-dev-tools/pages/usage/workspace-editor/#subcommand-generation) for those argument

Reply:
Thanks, we have created a custom command to create a node. So, I've modified it to include the pre-requisites in the example's description.